system_test/aest_sync_SUITE.erl

```diff
@@ -37,16 +37,14 @@
     name    => old_node1,
     peers   => [old_node2],
     backend => aest_docker,
-    % Change to a compatible fixed version when possible
-    source  => {pull, "aeternity/epoch:local"}
+    source  => {pull, "aeternity/epoch:v0.11.1"}
```

Comment:
If we agreed that new releases shall keep backwards compatibility of P2P network this may be updated to :latest though for now good enough.